xml/chapter5/section5/subsection2.xml

```diff
@@ -759,25 +795,18 @@ function compile_sequence(seq, target, linkage) {
     </SNIPPET>
     <SPLIT>
       <JAVASCRIPT>
-    Treating a return statement as if it is the last statement in a sequence
-    avoids compiling any
-    <QUOTE>dead code</QUOTE> after the return statement that can
+    Treating a return statement as if it were the last statement in a sequence
+    avoids compiling any <QUOTE>dead code</QUOTE> after the return statement that can
     never be executed.<FOOTNOTE>Our compiler does not detect all dead code. For example,
     a conditional statement whose consequent and alternative branches both end in a return
-    statement will not stop subsequent statements from being compiled. See exercises<SPACE/><REF NAME="ex:dead-code"/> and <REF NAME="ex:append_return_undefined"/>.
-    In the next section, we will see that we sometimes generate <EM>dead object code</EM>. For example,
-    we generate two branches for each function call depending on whether the function is compiled or primitive,
```

Comment:
This is really jumping the gun. The "dead object code" here is a tangent that is more distracting than helpful.